libbeat/docs/newbeat.asciidoc

```diff
@@ -128,10 +146,11 @@ beat_path [github.com/{github id}]:
 full_name [Firstname Lastname]: {Full Name}
 ---------
 
-This should now have created a directory `testbeat` inside our folder with several files. 
+You should see a directory called `testbeat` inside of your project folder. 
 
-We now have a raw template of the Beat, but still need to fetch the dependencies and setup the git repository.
+You now have a raw template of the Beat, but you still need to fetch the dependencies and set up the git repository.
 
+//REVIEWERS: We don't describe how to fetch dependencies or set up the git repo anywhere in the docs. Have we decided to skip talking about the make command in the docs because we want to remove  dependencies like python and the virtualenv? I'm not entirely opposed to pointing off to Jongmin's blog for details about running make and setting up the git repo, but if we do that, we need to be more explicit and we need to agree to update the blog entry whenever it gets out-of-date.  
```

Comment:
Yes, I didn't include it as it was part of the blog post, but we can get the content from the blog post and adjust it a bit to fit in the dev guide.